scripts/init_venv.py

```diff
@@ -134,20 +132,37 @@ def link_pyqt():
         link_name = os.path.join(venv_path, fn)
         if not os.path.exists(source):
             raise FileNotFoundError(source)
+        if os.path.exists(link_name):
+            os.unlink(link_name)
         print('{} -> {}'.format(source, link_name))
         os.symlink(source, link_name)
 
 
 def create_venv():
-    """Create a new virtualenv."""
-    utils.print_title("Creating virtualenv")
-    if os.name == 'nt':
-        sys_site = ['--system-site-packages']
-    else:
-        sys_site = []
-    subprocess.check_call(['virtualenv'] + sys_site +
-                          ['-p', sys.executable, g_path])
+    """Create a new venv."""
+    utils.print_title("Creating venv")
+
+    system_site_packages = True
```

Comment:
I'd prefer not using `system_site_packages` if possible (i.e. on non-Windows systems). On some of my test systems I have related libraries installed both system-wide and in the virtualenv, and I want to be certain only the one in the venv is used.

I believe you can simply call `venv_python('-m', 'pip', 'install', 'setuptools')` before `venv_python('setup.py', 'develop')` to install setuptools, and then it should work.